RetailCoder.VBE/UI/ToDoItems/ToDoExplorerDockablePresenter.cs

```diff
@@ -59,9 +62,9 @@ private void RefreshToDoList(object sender, EventArgs e)
 
         private void SortColumn(object sender, DataGridViewCellMouseEventArgs e)
         {
-            var columnName = Control.GridView.Columns[e.ColumnIndex].Name;
+            var columnName = _view.GridView.Columns[e.ColumnIndex].Name;
 
-            Control.TodoItems = _gridViewSort.Sort(Control.TodoItems, columnName);
+            _view.TodoItems = _gridViewSort.Sort(_view.TodoItems, columnName);
```

Comment:
I need to undo this and put it back to using the base reference to the control. I undid it because the code couldn't cast a mock to a `UserControl`. I need to go back and make the mock a `Mock<UserControl>` that implements `ITodoWindowView`.